workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePage/WorkflowResult.tsx

```diff
@@ -81,79 +81,124 @@ const ResultMessage = ({
   status,
   error,
   resultMessage,
+  executionSummary: executionSummary,
 }: {
   status?: ProcessInstanceStatusDTO;
   error?: ProcessInstanceErrorDTO;
   resultMessage?: WorkflowResultDTO['message'];
+  executionSummary?: string[];
 }) => {
   const errorMessage = error?.message || error?.toString();
+  const executionSummaryArray: string[] = executionSummary ?? [];
 
-  let statusComponent: ReactNode = <></>;
+  const getTimeFromExecutionSummary = (
+    keyword: 'started' | 'failed' | 'retriggered' | 'waiting' | 'completed',
+  ): string[] => {
+    const matchingMessage = executionSummaryArray.find(str =>
+      str.includes(keyword),
+    );
+    if (!matchingMessage) return [''];
+
+    const timeMatch = matchingMessage.match(
+      /(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(?:\.\d+)?Z)/,
```

Comment:
feel free to ignore: an example message as a comment here will help future maintenance.

Reply:
Does the rocket emoji mean that there will be a comment added? :-)